internal/backends/testing/slint_systest.proto

```diff
@@ -104,6 +104,27 @@ message ElementTypeNameAndId {
     string id = 2;
 }
 
+// Copied from enums.rs - can't be auto-generated :(
```

Comment:
It would be possible to do that in a build.rs still, i guess, if we wanted.
In the absence of that, is it possible to have a test that they are kept in sync?

Reply:
I don't know how to do that :( the rust enum is non_exhaustive and originates from core crate. What I really want is the match in the rust code to not have a `_ => { ... }` clause...

But meanwhile, I guess generating this in build.rs (along with conversion code) is probably the cleanest. I'll do that.

Reply:
Good idea about strum. Implemented that now :)